### PR TITLE
T6784: enabled repo-sync wokrflow only for current and equuleus

### PR DESCRIPTION
**Pull request by kumvijaya**
## Change Summary

T6784: enabled repo-sync wokrflow only for current and equuleus

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [ ] Other (please describe):

## Related Task(s)

https://vyos.dev/T6784

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [ ] I have linked this PR to one or more Phabricator Task(s)
- [ ] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Comment by kumvijaya** [comment]
@MergifyIo backport sagitta

**Comment by kumvijaya** [comment]
@MergifyIo backport equuleus

**Comment by kumvijaya** [comment]
@MergifyIo backport circinus

**Comment by kumvijaya** [comment]
@MergifyIo backport crux

**Review by c-po — APPROVED** [review]

File: .github/workflows/repo-sync.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ on:
         - closed
       branches:
         - current
+        - equuleus
```
Comment (sever-sever) [inline .github/workflows/repo-sync.yml:9]: But we have it in equuleus branch https://github.com/vyos/vyos-1x/blob/a79b59070ca0039aec0a43a68e427b1d75b40cd0/.github/workflows/repo-sync.yml#L8

Comment (kumvijaya) [reply .github/workflows/repo-sync.yml:9]: > But we have it in equuleus branch
> 
> https://github.com/vyos/vyos-1x/blob/a79b59070ca0039aec0a43a68e427b1d75b40cd0/.github/workflows/repo-sync.yml#L8

added backport for this branch

Also removed crux backport

**Review by sever-sever — APPROVED** [review]

**Pull request merged**